lametro/models.py

```diff
@@ -639,9 +639,14 @@ def _potentially_current_meetings(cls):
         six_hours_ago = cls._time_ago(hours=6)
         five_minutes_from_now = cls._time_from_now(minutes=5)
 
+        was_cancelled = Q(status="cancelled")
+        has_passed = Q(broadcast__observed=True) & ~Q(
+            pk__in=cls._streaming_meeting().values_list("pk")
```

Comment:
This is neat, first time I've seen `values_list()`. So from the docs, it looks like this is a way to list out all the values in a field. And you're using that to ensure that meetings that are currently streaming aren't being filtered out, cool!

Reply:
I love a values list. It's cuter (and more efficient) than making a list of values in a list comprehension over an entire queryset.